mav_local_planner/include/mav_local_planner/mav_local_planner.h

```diff
@@ -27,6 +27,7 @@
 #include <voxblox_loco_planner/voxblox_loco_planner.h>
 #include <voxblox_ros/esdf_server.h>
 #include <voxblox_planning_msgs/YawPolicyService.h>
+#include <voxblox_planning_msgs/SmootherNameService.h>
```

Comment:
The srv is exactly the same as the Yaw policy srv. I would just make one generic srv, which is independent of the thing you are changing.

Reply:
Changed.